react/Columns/Columns.less

```diff
@@ -10,15 +10,12 @@
   @media @desktop {
     flex-basis: 0;
     flex-grow: 1;
-
-    min-width: 0;
```

Comment:
This `min-width` looks like it is required to support the flexible `Columns` work implemented in https://github.com/seek-oss/seek-style-guide/pull/350

Reply:
Nice catch 👏